pages/volunteer_role.tsx

```diff
@@ -109,32 +108,32 @@ const VolunteerRolePage = () => {
           </Box>
         ) : null}
         <>
-        {roleData.location ? (
-          <Box
-          sx={{
-            typography: 'bodyLarge',
-            fontWeight: 'bold',
-            lineHeight: '0.5rem',
-            mt: '1rem',
-          }}
-          >
-            {'Location: '}
-            <span style={{ fontWeight: 'normal' }}>{roleData.location}</span>
-          </Box>
-        ) : null}
-        {roleData.duration ? (
-          <Box
-          sx={{
-            typography: 'bodyLarge',
-            fontWeight: 'bold',
-            lineHeight: '0.5rem',
-            mb: '1rem',
-          }}
-          >
-            {'Duration: '}
-            <span style={{ fontWeight: 'normal' }}>{roleData.duration}</span>
-          </Box>
-        ) : null}
+          {roleData.location ? (
```

Comment:
Shouldn't the prettier and jslint configs located in the root folder apply identical formatting rules for each contributor? It appears that there are big formatting related updates with some pull requests, and I'm wondering if individual contributors need to do anything to ensure we are all using the same formatting rules.

Reply:
Just confirming that  the following is used for .vscode/settings.json

```
{
  "eslint.validate": ["javascript", "typescript", "typescriptreact"],
  "editor.defaultFormatter": "esbenp.prettier-vscode",
  "editor.formatOnSave": true,
  "editor.codeActionsOnSave": {
    "source.fixAll.eslint": true
  },
  "editor.insertSpaces": true,
  "editor.tabSize": 2,
  "editor.rulers": [80],
  "prettier.singleQuote": true,
  "prettier.semi": false,
  "prettier.trailingComma": "all"
}
```